doc/cli/npm-update.md

```diff
@@ -18,6 +20,15 @@ packages.
 If no package name is specified, all packages in the specified location (global
 or local) will be updated.
 
+## WARNING
```

Comment:
Split this out into a separate patch, plz -- at some point we're going to need to update the docs again to fix this, and it's probably going to be forgotten unless we have some way of tracking that on, say, #6247.